curriculum/challenges/english/25-front-end-development/lab-movie-review-page/67ef8f695db52583424bd118.md

```diff
@@ -0,0 +1,179 @@
+---
+id: 67ef8f695db52583424bd118
+title: Design a Movie Review Page
+challengeType: 14
+dashedName: design-a-movie-review-page
+demoType: onClick
+---
+
+# --description--
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. Your should have a `main` element.
+2. Inside the `main` element, you should have an `h1` element for the movie title.
+3. Below the `h1` element, you should have an `img` element displaying the movie cover with a descriptive `alt` attribute describing the image.
+4. You should have a `p` element containing a brief movie description.
+5. You should have another `p` element to display the movie rating. It should contain a `strong` element with the text `Movie Rating`.
+6. You should have a `span` element with an `aria-hidden` attribute set to `true` containing a visual representation of the rating using stars `⭐⭐⭐⭐⭐⭐⭐⭐⭐☆` and a numerical value in parentheses (e.g. `9.2/10`).  
```

Comment:
maybe the stars is better given as unicode character? &#x2B50; `&#x2B50;` and &#9734; `&#9734;`

Reply:
from the review in prototype stage they agreed to stick with the stars as is.